actions/docs-verifier/src/ActionRunner/Program.cs

```diff
@@ -67,7 +67,8 @@
         ? file.PreviousFileName
         : (file.IsRemoved() ? file.FileName : null);
 
-    bool isDeletedToc = deletedFileName is not null && deletedFileName.Equals("toc.yml", StringComparison.OrdinalIgnoreCase);
+    bool isDeletedToc = deletedFileName is not null
+        && deletedFileName.EndsWith("toc.yml", StringComparison.OrdinalIgnoreCase);
```

Comment:
This may have false negatives if a landing page file (which needs redirection) ends with "toc".

Reply:
If it ends with `toc.yml` that is a problem